images/minio/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM minio/minio:RELEASE.2018-04-19T22-54-58Z
```

Comment:
we set `MINIO_BASE_IMAGE` back in `images/Makefile` but we aren't using that here, we are just hard coding the from/base image.  We should also consider injecting the tag here with a build arg (see https://github.com/rook/rook/blob/master/images/ceph/Makefile#L43 for an example)